Traceback (most recent call last):
  File "E:\Learning_material\junior\AI\5_FinalLab\FinalLab_ai\main.py", line 211, in <module>
    main()
  File "E:\Learning_material\junior\AI\5_FinalLab\FinalLab_ai\main.py", line 203, in main
    train_loader, val_loader = load_data()
                               ^^^^^^^^^^^
  File "E:\Learning_material\junior\AI\5_FinalLab\FinalLab_ai\main.py", line 50, in load_data
    with open(TRAIN_FILE, 'r') as file:
         ^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/kaggle/working/dataset\\train.txt'
